lib/index.js

```diff
@@ -71,9 +50,9 @@ function getVNodeType(t, type) {
 		flags = VNodeFlags.ComponentUnknown;
 	}
 	return {
-		type,
+		type: type,
 		isComponent: component,
```

Comment:
To keep this consistent we should rename `component` to `isComponent` for readability.